src/pages/List.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './pages.css';
+
+export default function List() {
+  return (
+    <div>
```

Comment:
Let’s practice better semantics and change these `<div>` tags to `<main>` tags. Thanks!